shared/teams/add-members-wizard/confirm.tsx

```diff
@@ -0,0 +1,213 @@
+  const onBack = () => dispatch(nav.safeNavigateUpPayload())
+
+  return (
+    <Kb.Modal
+      onClose={onClose}
+      allowOverflow={true}
+      mode="DefaultFullHeight"
+      header={{
+        leftButton: <Kb.Icon type="iconfont-arrow-left" onClick={onBack} />,
+        title: <ModalTitle teamname={teamname} title={`Inviting ${addingMembers.length} ${noun}`} />,
+      }}
+      footer={{
+        content: <Kb.Button fullWidth={true} label={`Invite ${addingMembers.length} ${noun} & finish`} />,
+      }}
+    >
+      <Kb.Box2 direction="vertical" fullWidth={true} style={styles.body} gap="small">
+        <Kb.Box2 direction="vertical" fullWidth={true} gap="tiny">
+          <AddingMembers />
+          <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.controls}>
+            <AddMoreMembers />
+            <RoleSelector />
+          </Kb.Box2>
+        </Kb.Box2>
+        <Kb.Box2 direction="vertical" fullWidth={true} gap="xtiny">
+          <Kb.Text type="BodySmallSemibold">Join channels</Kb.Text>
+          <Kb.Box2 direction="vertical" fullWidth={true}>
+            <Kb.Text type="BodySmall">Your invitees will be added to 3 channels.</Kb.Text>
+            <Kb.Text type="BodySmall">
+              {/* TODO: Hook this up when default channels settings are hooked up */}
+              <Kb.Text type="BodySmallBold">#general</Kb.Text>,{' '}
+              <Kb.Text type="BodySmallBold">#random</Kb.Text>, and{' '}
+              <Kb.Text type="BodySmallBold">#hellos</Kb.Text>.{' '}
+              <Kb.Text type="BodySmallPrimaryLink">Change this</Kb.Text>
+            </Kb.Text>
+          </Kb.Box2>
+        </Kb.Box2>
+      </Kb.Box2>
+    </Kb.Modal>
+  )
+}
+
+const AddMoreMembers = () => {
+  const {showingPopup, toggleShowingPopup, popup, popupAnchor} = Kb.usePopup(getAttachmentRef => (
+    <Kb.FloatingMenu
+      attachTo={getAttachmentRef}
+      closeOnSelect={true}
+      onHidden={toggleShowingPopup}
+      visible={showingPopup}
+      items={[{title: 'From Keybase'}, {title: 'By email address'}, {title: 'By phone number'}]}
+    />
+  ))
+  return (
+    <>
+      <Kb.Button
+        mode="Secondary"
+        small={true}
+        label="Add people"
+        onClick={toggleShowingPopup}
+        ref={popupAnchor}
+      />
+      {popup}
+    </>
+  )
+}
+
+const RoleSelector = () => {
+  const dispatch = Container.useDispatch()
+  const [showingMenu, setShowingMenu] = React.useState(false)
+  const storeRole = Container.useSelector(s => s.teams.addMembersWizard.role)
+  const [role, setRole] = React.useState(storeRole)
+  const onSelectRole = newRole => setRole(newRole)
+  const onConfirmRole = newRole => {
+    setRole(newRole)
+    setShowingMenu(false)
+    dispatch(TeamsGen.createSetAddMembersWizardRole({role: newRole}))
+  }
+  return (
+    <Kb.Box2 direction="horizontal" gap="tiny" alignItems="center">
+      <Kb.Text type="BodySmall">Invite as: </Kb.Text>
+      <FloatingRolePicker
```

Comment:
There's a prop on this to change the text from e.g. 'Make owner' to "Add as owners" I think.

Reply:
Fixed in #22980